src/pinns_pde_solve.jl

```diff
@@ -266,11 +266,27 @@ function get_loss_function(loss_function, train_set)
         sum(loss_function(phi, x, θ, derivative))
     end
 
-    function loss(θ, phi, derivative)
-        if (loss_function isa Array)
-            1.0f0/τ *sum(sum(abs2,inner_loss(l,phi,x,θ,derivative) for x in set) for (l,set) in zip(loss_function,train_set))
+    if !(loss_function isa Array)
+        loss_function = [loss_function]
+        train_set = [train_set]
+    end
+
+    function loss(θ, phi, derivative, stochastic_loss)
```

Comment:
should probably make this dispatching already , since we will want to have a whole set of training strategies.